pkg/sidecar/sidecar.go

```diff
@@ -17,59 +17,96 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/andres-erbsen/clock"
+	proto "github.com/spiffe/go-spiffe/proto/spiffe/workload"
 	"github.com/spiffe/spire/api/workload"
-	proto "github.com/spiffe/spire/proto/api/workload"
+	"golang.org/x/sys/unix"
 )
 
-// sidecar is the component that consumes the Workload API and renews certs
+// Config contains config variables when creating a SPIFFE Sidecar.
+type Config struct {
+	AgentAddress       string
```

Comment:
hcl struct tags that were left in ParseConfig when the unnamed struct was extracted should be restored here

Reply:
done